src/Artsy/Router/buildServerApp.tsx

```diff
@@ -134,8 +136,27 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
           scripts: scripts.join("\n"),
         })
       } catch (error) {
-        console.error("[Artsy/Router/buildServerApp] Error:", error)
-        reject(error)
+        if (
+          error instanceof RRNLRequestError &&
+          error.res &&
+          error.res.errors &&
+          error.res.errors.length === 1 &&
+          error.res.errors[0].extensions &&
+          error.res.errors[0].extensions.httpStatusCodes &&
+          error.res.errors[0].extensions.httpStatusCodes.length === 1
```

Comment:
This can probably use our `get` utility.

Reply:
Heads up that per a conversation in Slack, I'm going to pick this PR up - I'll write some tests, and convert this to use `get`.